CONTRIBUTING.md

```diff
@@ -30,23 +30,10 @@ Do you want to **contribute a failing test**? [This tutorial will show you how](
 
 ## Preparing Pull Request
 
-3 steps will make your pull-request easy to merge:
-
 - **1 feature per pull-request**
 - **new features need tests**
-- CI must pass... you can mimic it locally by running
-
-    ```bash
-    composer complete-check
-    ```
-
-- Do you need to fix coding standards?
-
-    ```bash
-    composer fix-cs
```

Comment:
The script is exists, contributing is to rector-src which the composer.json consist the script, see 

https://github.com/rectorphp/rector-src/blob/73aa7e3d68d74fc0b9b26fc543bf1207cfc8d0dd/composer.json#L126